From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Sun, 9 Aug 2020 00:34:37 +0000
Subject: [PATCH] pam-arch: Replace pam_tally with pam_faillock

pam 1.4.0 removed the former and replaces it with the latter.

https://bugs.archlinux.org/task/67485
---
 data/pam-arch/gdm-fingerprint.pam | 2 +-
 data/pam-arch/gdm-smartcard.pam   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/pam-arch/gdm-fingerprint.pam b/data/pam-arch/gdm-fingerprint.pam
index a4808617..57d57925 100644
--- a/data/pam-arch/gdm-fingerprint.pam
+++ b/data/pam-arch/gdm-fingerprint.pam
@@ -1,4 +1,4 @@
-auth     required  pam_tally.so onerr=succeed file=/var/log/faillog
+auth     required  pam_faillock.so onerr=succeed file=/var/log/tallylog
 auth     required  pam_shells.so
 auth     requisite pam_nologin.so
 auth     required  pam_env.so
diff --git a/data/pam-arch/gdm-smartcard.pam b/data/pam-arch/gdm-smartcard.pam
index ec6f75d5..0852476a 100644
--- a/data/pam-arch/gdm-smartcard.pam
+++ b/data/pam-arch/gdm-smartcard.pam
@@ -1,4 +1,4 @@
-auth     required  pam_tally.so onerr=succeed file=/var/log/faillog
+auth     required  pam_faillock.so onerr=succeed file=/var/log/tallylog
 auth     required  pam_shells.so
 auth     requisite pam_nologin.so
 auth     required  pam_env.so
